var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -22,8 +22,13 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
     maintainers("davidbeckingsale", "adrienbernede")
 
     license("MIT")
-
```

Comment:
Keep this extra line between the license and all the versions.

Reply:
Done